packages/api/tests/lambdas/test-publish-granules.js

```diff
@@ -103,3 +108,37 @@ test.serial('The publish-granules Lambda function takes a DynamoDB stream event
 
   t.is(Messages.length, 2);
 });
+
+test.serial('The publish-granules Lambda function takes a DynamoDB stream event with a REMOVE record and adds a deletedAt to the SNS message', async (t) => {
+  const { QueueUrl } = t.context;
+
+  const granuleId = randomString();
+
+  const event = {
+    Records: [
+      {
+        dynamodb: {
+          OldImage: {
+            granuleId: { S: granuleId },
+            status: { S: 'running' }
+          }
+        },
+        eventName: 'REMOVE'
+      }
+    ]
+  };
+
+  await handler(event);
+
+  const { Messages } = await sqs().receiveMessage({ QueueUrl, WaitTimeSeconds: 10 }).promise();
+
+  t.is(Messages.length, 1);
+
+  const snsMessage = JSON.parse(Messages[0].Body);
+  const message = JSON.parse(snsMessage.Message);
+  const granuleRecord = message.record;
+
+  t.is(granuleRecord.granuleId, granuleId);
+  t.is(!!granuleRecord.deletedAt, true);
```

Comment:
We should change this to 

```
t.true(typeof granuleRecord.deletedAt, 'number')
```

It tests for the existence of the value and that it is the correct type

Reply:
yes